app/src/main/java/com/byeboo/app/presentation/splash/OnBoardingViewModel.kt

```
@@ -0,0 +1,70 @@
+    )
+
+    fun nextPage() {
+        if (_pageIndex.intValue < pages.lastIndex) {
+            _pageIndex.intValue += 1
+        }
+    }
+
+
+    fun skipPage() {
+
+    }
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)